### PR TITLE
Add help text for affiliations model

### LINKED ISSUE #176: Clarify people "affiliations"
At the bottom of the "people" model admin, we've got a section for "affiliations".
image

What useful text can we put there to make its purpose clearer?

### PR DESCRIPTION
**Pull request by alanmoo**
Slight hack, @gideonthomas said it's the most reasonable way though. Closes #176

### CODE REVIEW

**Comment by gideonthomas** [comment]
@alanmoo I would say change the `verbose_name` in the `InlineAdmin` instead if you're just updating the heading.

**Comment by alanmoo** [comment]
Does that show up anywhere else where it might be obnoxiously long?

File: app/networkapi/static/people/js/admin.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ function run($) {
     $(".field-quote").toggle(this.checked);
     $(".field-bio").toggle(this.checked);
   });
+
+   $('#affiliations-group>h2').append(' (e.g. companies, nonprofits, universities, projects and other organizations this person is actively involved with)');
```
Comment (Pomax) [inline app/networkapi/static/people/js/admin.js]: indent.

**Comment by Pomax** [comment]
I'm inclined to agree with @gideonthomas here - this is a little too hacky, let's change it in the model and then we can see if that causes issues anywhere. If so, we can address those issues with a clean and simple css `text-overflow: ellipsis`

**Comment by gideonthomas** [comment]
It should only show up where you use the inline. Since we only use the inline in the People Admin, I think it's safe

**Comment by gideonthomas** [comment]
@alanmoo don't change it in the model. Change it in the InlineAdmin here:
https://github.com/mozilla/network-api/blob/master/app/networkapi/people/admin.py#L12-L14

You should just be able to do:
```python
class AffiliationAdmin(admin.TabularInline):
    model = Affiliation
    verbose_name = 'Blah blah blah'
    extra = 1
```

**Comment by alanmoo** [comment]
What are you talking about that's exactly what I did and definitely didn't amend my commit again Github why are you saying I did

**Pull request merged**